index.js

```diff
@@ -13,71 +7,12 @@ function ask(questionText) {
   });
 }
 
-function findMiddlePoint(min, max) {
-  let middle = Math.floor((min + max) / 2)
-  return middle;
-}
-
-
 start();
 
 async function start() {
-
-  console.log("Let's play a game where you (human) make up a number between 1 and 100 and I (computer) try to guess it.")
+  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-
-  let guess = await ask("is it " + guessnum + "?\n");
-
-
-
-
-  if (guess.toLowerCase() === 'n' || guess.toLowerCase() === 'no') {
-
-    guessing = true;
-    while (guessing === true) {
-      let highLow = await ask('Is it higher (H), or lower (L)?');
-      if (highLow.toUpperCase() === 'H') {
-        min = guessnum + 1
-        guessnum = findMiddlePoint(min, max);
-        guess = await ask("is it " + guessnum + "?\n");
-        guessCount += 1
-
-        if (guess.toLowerCase() === 'y' || guess.toLowerCase() === 'yes') {
-          guessing = false
-        }
-      } else if (highLow.toUpperCase() === 'L'){
-        max = guessnum - 1
-        guessnum = findMiddlePoint(min, max);
-        guess = await ask("is it " + guessnum + "?\n");
-        guessCount += 1
-        if (guess.toLowerCase() === 'y' || guess.toLowerCase() === 'yes') {
-          guessing = false
-        }
-      } else {
-        console.log("You did not answer my question! I will let it slide this time, try again.")
-      }
-    }
-  }
-
-  console.log('Your number was ' + guessnum + '!\nI guessed it in ' + guessCount + ' tries.')
-
+  // Now try and complete the program.
   process.exit();
-}
-
-
-//To-Do
-//fix bug if niether y or n is entered
```

Comment:
add an `else` that matches up to your `if` on line 35 to handle this (you may also need an `else if` to handle if the answer is yes)